apps/frontend/core-dapp/src/features/portfolio/PositionsAndHistoryNext.tsx

```diff
@@ -0,0 +1,100 @@
+import { t } from '@lingui/macro'
+import { Box, centered, media, spacingIncrement } from 'prepo-ui'
+import { useState } from 'react'
+import styled, { useTheme } from 'styled-components'
+import FilterButton from '../../components/Filter/FilterButton'
+import Tabs from '../../components/Tabs'
+import History from '../history/History'
+import Positions from '../position/PositionsNext'
+
+const StyledTabs = styled(Tabs)`
+  &&&& {
+    .ant-select-selection-item {
+      padding-right: 20px;
+    }
+
+    .ant-select-selection-item,
+    .ant-tabs-tab-btn {
+      font-size: ${({ theme }): string => theme.fontSize.lg};
+    }
+
+    .ant-select-arrow {
+      left: 100%;
+      padding: 0 ${spacingIncrement(6)};
+      transform: translate(-100%, -50%);
+    }
+
+    .ant-tabs-nav {
+      flex-direction: column;
+      margin-bottom: 0;
+      ${media.desktop`
+        flex-direction: row;
+      `}
+    }
+
```

Comment:
Add same padding as figma (note that we have to add in .ant-tabs-nav-wrap to prevent ink bar from overflow-hidden

```suggestion

    .ant-tabs-nav-wrap {
      padding: ${spacingIncrement(16)};
      padding-bottom: ${spacingIncrement(8)};
    }

```